godtools/App/Features/SetupParallel/SetupParallelLanguageView.swift

```diff
@@ -45,63 +45,66 @@ class SetupParallelLanguageView: UIViewController {
         
         super.viewDidLoad()
         
+        tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleSelectLanguageTapped))
+        tapRecognizer.cancelsTouchesInView = false
+        tapRecognizer.delegate = self
+        self.view.addGestureRecognizer(tapRecognizer)
```

Comment:
I noticed when I tapped No, the language list appeared and then the entire modal was dismissed.  It looks like because this tap gesture is being applied to the entire view.  Instead it should be applied to the selectLanguageButtonView.

Reply:
I noticed that at one point, but I thought it went away.  I can try to make that change.